protocol/Makefile

```diff
@@ -33,18 +33,13 @@ js/flow-types.js: build-stamp | config
 	node --harmony ./bin/flow.js
 	cp ./js/flow-types* ../shared/constants/types
 
-js/keybase_v1.js: build-stamp | config
```

Comment:
the enums have moved to flow-types. Also reduced the hierarchy a level so instead of

```
common.StatusCodes.scgeneric
```

you do

```
CommonStatusCodes.scgeneric
```